sigar_openbsd.go

```diff
@@ -282,7 +292,7 @@ func (self *Swap) Get() error {
 
 	rnswap := C.swapctl(C.SWAP_STATS, unsafe.Pointer(&swdev[0]), nswap)
 	if rnswap == 0 {
-		return nil
+		return fmt.Errorf("Failed to call swapctl(2)")
```

Comment:
It won't return an error in such case; I've ran `swapctl -d /dev/sd0b` to remove my `/dev/sd0b` as a swap device and this conditional didn't get triggered.